pandas/_libs/tslibs/timezones.pyx

```diff
@@ -316,10 +316,10 @@ cpdef bint tz_compare(object start, object end):
     return get_timezone(start) == get_timezone(end)
 
 
-cpdef tz_normalize(object tz):
+cpdef tz_standardize(object tz):
```

Comment:
Does this need to be a `cpdef`? Can it just be `cdef`? I don't know why end users would need to be able to do this.

Reply:
This function is called within a python file (`pandas/core/indexes/datetimelike.py`), so it can't just be a `cdef`